### PR TITLE
Version 1.1.0 release updates

### PR DESCRIPTION
**Pull request by timmyc**
Prepping for the release of v1.1.0

### CODE REVIEW

**Review by psealock — APPROVED** [review]
Changes look good to me ✅

File: readme.txt [hunk 1]
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 
 == Changelog ==
 
+= 1.1.0 TBD =
+- Tweak: Update form example to better format the form return data #3706
+- Dev: Align WooCommerce javascript code standards/linting/styles with WordPress core. #3674
```
Comment (jeffstieler) [inline readme.txt:76]: Omit this?

File: readme.txt [hunk 2]
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 
 == Changelog ==
 
+= 1.1.0 TBD =
+- Tweak: Update form example to better format the form return data #3706
```
Comment (jeffstieler) [inline readme.txt:75]: Omit this? It's only in the DevDocs..

File: readme.txt [hunk 3]
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 
 == Changelog ==
 
+= 1.1.0 TBD =
+- Tweak: Update form example to better format the form return data #3706
+- Dev: Align WooCommerce javascript code standards/linting/styles with WordPress core. #3674
+- Tweak: Added link to "go shopping" button #3712
+- Fix: Make analytics tables use the site's date format setting #3715
+- Fix: Alignment of select text #3723 🎉 edmundcwm
+- Dev: Added Storybook for documenting internal components. #3679
```
Comment (jeffstieler) [inline readme.txt:80]: Omit?

File: readme.txt [hunk 4]
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
...
+- Tweak: Update form example to better format the form return data #3706
+- Dev: Align WooCommerce javascript code standards/linting/styles with WordPress core. #3674
+- Tweak: Added link to "go shopping" button #3712
+- Fix: Make analytics tables use the site's date format setting #3715
+- Fix: Alignment of select text #3723 🎉 edmundcwm
+- Dev: Added Storybook for documenting internal components. #3679
+- Dev: Add PayFast payment gateway option for sites in South Africa #3738 
+- Dev: Onboarding: Update screen order and remove Jetpack connection in profiler #3739
+- Dev: Onboarding - business step: add more options in the competitors list and other fixes #3812
+- Dev: Onboarding: Redesign plugin benefits screen #3764
```
Comment (jeffstieler) [inline readme.txt:84]: These 4 lines aren't actually "Dev". They are either "Enhancement" or "Tweak".

File: readme.txt [hunk 5]
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
...
+- Fix: inconsistent wording downloads report #3844 🎉 jobthomas
+- Tweak: Onboarding: Use full width template for homepage in stores using Storefront #3846
+- Tweak: Remove 'add first product' note. #3876
+- Enhancement: Improve focus on task list #3796
+- Enhancement: Allow individual payment method setup in the onboarding task list #3782
+- Dev: Onboarding: Add toggles to configured payments in task list #3801
```
Comment (jeffstieler) [inline readme.txt:90]: Again, not dev.

File: readme.txt [hunk 6]
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
...
+- Enhancement: "Personalize your store" reminder: new inbox notification #3895
+- Fix: Onboarding: only validate other platform name for 'other' selling venues #4011
+- Performance: only query necessary data in Orders Panel. #3969
+- Tweak: Remove icon-button classes from Tag component #3993
+- Enhancement: WooCommerce Shipping order page banner prompt #3955
+- Performance: speed up indicators endpoint response. #3997
+- Enahncement: Add WC Pay to startup checklist
+- Enhancement: Marketing Tab
+- Enhancement: Add WooCommerce Payments Inbox note #4013
+- Dev: Added hook to delete woocommerce_onboarding_homepage_post_id #4015
```
Comment (jeffstieler) [inline readme.txt:110]: Not dev, probably "fix"?

**Review by octaedro — APPROVED** [review]
It's testing well here :shipit:

**Comment by timmyc** [comment]
@jeffstieler is it okay if I merge this as-is, then apply those readme changes in a follow-up PR? Now that I've been cherry-picking some fixes to `release\1.1.0` I fear that things might get tricky with the branch.

**Comment by jeffstieler** [comment]
> is it okay if I merge this as-is, then apply those readme changes in a follow-up PR?

With the cherry picking, I don't think we need to (or should) merge this branch into `master`. While this comparison shows a diff, these commits are already on the `master` branch: https://github.com/woocommerce/woocommerce-admin/commits/master

**Comment by psealock** [comment]
>With the cherry picking, I don't think we need to (or should) merge this branch into master. While this comparison shows a diff, these commits are already on the master branch: https://github.com/woocommerce/woocommerce-admin/commits/master

That works for me, but the version bump and changelog changes need to make their way into master.

**Comment by timmyc** [comment]
Closing in favor of #4158

**Pull request closed without merging**